packages/wallet/src/contract-tests/test-utils.ts

```diff
@@ -1,67 +1,38 @@
-import {ethers} from "ethers";
-import {CommitmentType, Commitment, signCommitment2} from "../domain";
+import { ethers } from "ethers";
+import { CommitmentType, Commitment, signCommitment2 } from "../domain";
 import {
   createDepositTransaction,
   createForceMoveTransaction,
   createConcludeTransaction,
   createRefuteTransaction,
-  createRespondWithMoveTransaction
+  createRespondWithMoveTransaction,
 } from "../utils/transaction-generator";
-import loadJsonFile from 'load-json-file';
-import path from 'path';
-import {signCommitment} from "../domain";
-// TODO: Make sure this import works in the scenario where build/contracts is empty
-// so that the webpack and tsc build for  @statechannels/wallet still passes
-import TrivialAppArtifact from '../../build/contracts/TrivialApp.json';
-import {bigNumberify} from "ethers/utils";
-import {channelID, Channel} from "fmg-core/lib/channel";
-import {ADJUDICATOR_ADDRESS, ETH_ASSET_HOLDER_ADDRESS} from "../constants";
-import {ADDRESS_ZERO} from "fmg-core";
+import { signCommitment } from "../domain";
+
+import { bigNumberify } from "ethers/utils";
+import { channelID, Channel } from "fmg-core/lib/channel";
+import { ADJUDICATOR_ADDRESS, ETH_ASSET_HOLDER_ADDRESS } from "../constants";
+import { ADDRESS_ZERO } from "fmg-core";
 import { JsonRpcProvider, TransactionRequest } from "ethers/providers";
+import { getLibraryAddress } from "../utils/contract-utils";
 
-let networkMap;
-export async function getLibraryAddress(networkId) {
-  networkMap = networkMap || await getNetworkMap();
-  if (networkMap[networkId] && networkMap[networkId][TrivialAppArtifact.contractName]) {
-    return networkMap[networkId][TrivialAppArtifact.contractName]
-  }
-  console.error(networkId, networkMap);
-  
-  throw new Error("Library missing")
-}
 export const fiveFive = [bigNumberify(5).toHexString(), bigNumberify(5).toHexString()] as [string, string];
 export const fourSix = [bigNumberify(4).toHexString(), bigNumberify(6).toHexString()] as [string, string];
 
 export const defaultDepositAmount = fiveFive[0];
 
-export const getNetworkMap = async () => {
-  try {
-    return await loadJsonFile(path.join(__dirname, '../../deployment/network-map.json'));
-  } catch (err) {
-    if (!!err.message.match('ENOENT: no such file or directory')) {
-      return {};
-    } else {
-      throw err;
-    }
-  }
-};
-
 export async function getChannelId(provider, channelNonce, participantA, participantB) {
   const network = await provider.getNetwork();
   const networkId = network.chainId;
-  const libraryAddress = await getLibraryAddress(networkId);
+  const libraryAddress = await getLibraryAddress(networkId, "TrivialApp");
```

Comment:
nit: I wonder if thous should be like `TrivialAppArtifact.contractName`

Reply:
I think the advantage of this is that the typescript can be compiled without the artifacts being present.